iFixit/toolbox
# Add Grid component

This pull adds a ~`FormLayout`~ `Grid` component to Toolbox. This component can used to easily create ~responsive forms~ grid layouts.

[~`FormLayout`~`Grid` documentation](https://deploy-preview-6--toolbox.netlify.com/#!/Grid)
> **Tip:** Click the "Code" link below each example to view and edit the code for the live example.

QA
---
* Go to https://deploy-preview-6--toolbox.netlify.com/#!/Grid
* Assert that the examples look like this on desktop screen sizes:
![image](https://user-images.githubusercontent.com/4608155/29440672-b768c604-837b-11e7-927a-eb5a83d9b648.png)
* Assert that all the grid cells are full-width on mobile screen sizes
* Assert consistency across all browsers and devices
  * **Note:** In Safari, all the cells might be full-width even on desktop. This is expected.

Part of https://github.com/iFixit/ifixit/issues/20203
